### PR TITLE
Feat(CX-3369): redesign the swa landing page part 5

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3369]

### Description

Web
image

mWeb
image

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3369]: https://artsyproduct.atlassian.net/browse/CX-3369?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                <Flex justifyContent="space-between" mb={[0.5, 1.5]}>
+                  <Text variant={["lg-display", "xl"]} overflowEllipsis>
+                    {priceRealized?.display}
+                  </Text>
+                  <Text
+                    variant={["lg-display", "xl"]}
+                    color="green100"
+                    overflowEllipsis
+                  >
+                    123%
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx:78]: Where is this value from?

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                  123%
+                </Text>
+              </Flex>
+
+              <Text
+                variant={["xs", "sm-display"]}
+                color="black60"
+                overflowEllipsis
+              >
+                Estimate $100,000–100,000
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx:161]: Do we want to show values as placeholder?

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+      `}
+      placeholder={PLACEHOLDER}
+      render={({ props, error }) => {
+        if (error) {
+          console.error(error)
+          return null
+        }
+
+        if (!props || !props.recentlySoldArtworks) {
+          return PLACEHOLDER
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx:204]: You mean `return <PLACEHOLDER />` ?

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SoldArtworkDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+export const SoldArtworkDetailsFragmentContainer = createFragmentContainer(
+  SoldArtworkDetails,
+  {
+    artwork: graphql`
+      fragment SoldArtworkDetails_artwork on Artwork {
+        href
+        title
+        date
+        cultural_maker: culturalMaker
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SoldArtworkDetails.tsx:73]: Why snake case? Unless in cases of name conflicts or maybe (rarely) for use as Object property shorthand when sending params that are not converted to snake_case on Metaphysics for mutation, I'm not sure there is any benefit to this in the frontend. 🤷

**Review by lordkiz — COMMENTED** [review]
Looks Great. I am mainly concerned about the values in the placeholder.

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx:161]: same comment, maybe we can just have those as a gray/black box as in app

**Review by MounirDhahri — COMMENTED** [review]
LGTM, only comments I have are the same as @lordkiz

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx (hunk 1 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx:78]: Oops, i wanted to ask in the channel where to get the info from and didn't 👀 
wip

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx:161]: Oki, I'll use grey boxes

File: src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworkSaveButton.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  const [isSaved, setIsSaved] = useState(true)
+
+  const handleClick = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    event.preventDefault()
+    setIsSaved(!isSaved)
+
+    // TODO: Display modal window with collections
+    console.log("[debug]", artworkId, collectionId)
```
Comment (MounirDhahri) [inline src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworkSaveButton.tsx:22]: 👀

**Review by MounirDhahri — COMMENTED** [review]
LGTM, only one comment and we can get this merged. I believe Dasha doesn't want to have the save icon available in the rail and we should therefore delete it.

File: src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworkSaveButton.tsx (hunk 5 above)
Comment (dariakoko) [reply src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworkSaveButton.tsx]: Looks like i merged main incorrect, this is not my change
Merged again and the pr looks cleaner now

**Pull request merged**